redash/serializers.py

```diff
@@ -23,20 +23,27 @@ def public_widget(widget):
         'created_at': widget.created_at
     }
 
-    if widget.visualization and widget.visualization.id:
-        query_data = models.QueryResult.query.get(widget.visualization.query_rel.latest_query_data_id).to_dict()
+    v = widget.visualization
+    if v and v.id:
+        if v.query_rel.latest_query_data_id:
```

Comment:
We no longer need to do this -- public dashboard should use the results API as the rest of the pages. 👌

Reply:
🎉 fc7323de